File: Funproject/Funproject/CoreData/NSManagedObjects/Employee+CoreDataProperties.m
//
//  Employee+CoreDataProperties.m
//  Funproject
//
//  Created by Stig von der Ahé on 02/10/2024.
//
//

#import "Employee+CoreDataProperties.h"

@implementation Employee (CoreDataProperties)

+ (NSFetchRequest<Employee *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"Employee"];
}

@dynamic check_in_date_time;
@dynamic company;

@end

File: Funproject/Funproject/Models/CoreData/NSManagedObjects/Employee+CoreDataProperties.swift
//
//  Employee+CoreDataProperties.swift
//  Funproject
//
//  Created by Stig von der Ahé on 03/10/2024.
//
//

import Foundation
import CoreData


extension Employee {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Employee> {
        return NSFetchRequest<Employee>(entityName: "Employee")
    }

    @NSManaged public var check_in_date_time: String?
    @NSManaged public var createdAt: Date?
    @NSManaged public var company: Company?

}
